test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -91,9 +91,20 @@ public void testFailure(Failure failure) throws Exception {
         GradleMessageBuilder gradleMessageBuilder = new GradleMessageBuilder(b);
         gradleMessageBuilder.appendAllOpts(failure.getDescription());
 
+        if (isRestApiCompatibilityTest(task)) {
+            b.append(System.lineSeparator());
+            b.append(
+                "This is a Rest Api Compatibility Test. "
+                    + "See the developers guide for details how to troubleshoot - REST_API_COMPATIBILITY.md"
```

Comment:
Should we include an actual URL?

Reply:
you mean a github link ? https://github.com/elastic/elasticsearch/blob/master/REST_API_COMPATIBILITY.md
In that case it should be branch aware if we at some point change it?

Reply:
It's probably not worth being branch aware.